frontend/src/pages/HeadPage.tsx

```diff
@@ -0,0 +1,31 @@
+import {useNavigate} from "react-router-dom";
+
+
+export default function HeadPage(){
+    const navigate = useNavigate()
+
+    function personalDataOnClick(){
+        navigate("/user")
+    }
+
+    function exercisesOnClick(){
+        navigate("/exercises")
+    }
+
+    function weekPlanOnClick(){
+        navigate("/weekplan")
+    }
+
+    function calendarOnClick(){
+        navigate("/calendar")
+    }
+
+    return(
+        <div className={"Menu"}>
```

Comment:
Hier steht className Menu, aber woher kommt die Regel?
(ich sehe keinen CSS Import und falls das in einer globalen Datei enthalten ist, ist Menu vermutlich zu allgemein/ nicht spezifisch genug)